src/Allure.XUnit/Attributes/AllureAllureIdAttribute.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Allure.Xunit.Attributes
+{
+    /// <summary>
+    /// Internal AllureId of test case, used to permanent mapping result to existing automatic test case. NB: don't confuse with "test case id" aka id for external TMS system.
+    /// https://docs.qameta.io/allure-testops/ecosystem/intellij-plugin/#link-tests-in-ide-to-tests-cases-in-allure-testops-via-allureid
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public class AllureAllureIdAttribute : Attribute, IAllureInfo
```

Comment:
Can we use name `AllureIdAttribute`? `AllureAllureId` is slightly strange

Reply:
@neparij Do you agree with such naming?

Reply:
Personally I would accept any naming, just please choose anything that fits and accept MR

Reply:
@AzatKalim fixed.